### PR TITLE
Update detray version to v49

### PR DESCRIPTION
**Pull request by beomki-yeo**

### CODE REVIEW

**Comment by niermann999** [comment]
Since there are material maps in the geometry now, you will need an accessor that overloads on material slab vectors vs material grids, whenever you access the material store in a visitor, e.g.:
https://github.com/acts-project/detray/blob/fcb1c85ba8bdb8799b2f42a4bba37b1ef35858dd/core/include/detray/propagator/actors/pointwise_material_interactor.hpp#L74-L75

**Comment by beomki-yeo** [comment]
I don't think this is a good design. We should define accessors globally instead of doing this for every kernels

**Comment by beomki-yeo** [comment]
Detray vesion is bumped to 49

File: tests/common/tests/kalman_fitting_wire_chamber_test.hpp [hunk 1]
```diff
@@ -67,6 +67,7 @@ class KalmanFittingWireChamberTests : public KalmanFittingTests {
         vecmem::host_memory_resource host_mr;
 
         detray::wire_chamber_config wire_chamber_cfg;
+        wire_chamber_cfg.mapped_material(detray::vacuum<scalar>());
```
Comment (beomki-yeo) [inline tests/common/tests/kalman_fitting_wire_chamber_test.hpp:70]: @niermann999  2 GeV test of wire chamber still fails with (berylium + 100 um thickness) setup so I decided to make portal materials empty :/

Comment (niermann999) [reply tests/common/tests/kalman_fitting_wire_chamber_test.hpp:70]: Ok

**Review by niermann999 — APPROVED** [review]

**Pull request merged**